src/applications/personalization/profile/hooks/useNotificationSettingsUtils.js

```diff
@@ -192,17 +244,38 @@ export const useNotificationSettingsUtils = () => {
   const useFilteredItemsForMHVNotifications = itemIds =>
     useMemo(
       () => {
-        return toggles.showMhvNotificationSettings
-          ? itemIds
-          : itemIds.filter(itemId => {
-              return !BLOCKED_MHV_NOTIFICATION_IDS.includes(itemId);
-            });
+        // Always exclude these items: ticket# 89524
+        const alwaysExcludedItems = ['item7', 'item8', 'item11', 'item12'];
```

Comment:
isn't this the same as the BLOCKED_MHV_NOTIFICATION_IDS constant? I'm not sure why this was recreated here.

Reply:
You're correct! I forgot to replace it with `BLOCKED_MHV_NOTIFICATION_IDS` constant after I was done testing my hypothesis #whoops